src/ledger/LedgerTxn.cpp

```diff
@@ -991,18 +1080,26 @@ LedgerTxn::Impl::loadAllOffers(LedgerTxn& self)
 LedgerTxnEntry
 LedgerTxn::loadBestOffer(Asset const& buying, Asset const& selling)
 {
-    return getImpl()->loadBestOffer(*this, buying, selling);
+    return getImpl()->loadBestOffer(*this, buying, selling, nullptr);
+}
+
+LedgerTxnEntry
+LedgerTxn::loadBestOffer(Asset const& buying, Asset const& selling,
+                         LedgerKey const& previousBest)
```

Comment:
`previousBest` should just be a `LedgerKey const* previousBest` as you can deal with null